### PR TITLE
docs: update the security policy

### PR DESCRIPTION
**Pull request by raisedadead**
Updates the security policy to the new email and adds the PGP form on FlowCrypt

### CODE REVIEW

File: docs/security.md [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
 
-If you think you have found a vulnerability, _please report responsibly_. Don't create GitHub issues for security issues. Instead, please send an email to `security@freecodecamp.org` and we'll look into it immediately.
+> [!NOTE]
+> If you think you have found a vulnerability, **please report responsibly**. Do not create GitHub issues for security issues. Instead follow this guide.
 
-Ensure that you are using the **latest**, **stable** and **updated** version of the Operating System and Web Browser available to you on your machine.
+### Guidelines
 
-We appreciate any responsible disclosure of vulnerabilities that might impact the integrity of our platforms and users.
+We appreciate a responsible disclosure of vulnerabilities that might impact the integrity of our platforms and users. In interest of saving everyone time, we encourage you to report vulnerabilities with these in mind:
```
Comment (nhcarrigan) [inline docs/security.md:12]:
```suggestion
We appreciate a responsible disclosure of vulnerabilities that might impact the integrity of our platforms and users. In the interest of saving everyone time, we encourage you to report vulnerabilities with these in mind:
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
nit

File: docs/security.md [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
 
-If you think you have found a vulnerability, _please report responsibly_. Don't create GitHub issues for security issues. Instead, please send an email to `security@freecodecamp.org` and we'll look into it immediately.
+> [!NOTE]
+> If you think you have found a vulnerability, **please report responsibly**. Do not create GitHub issues for security issues. Instead follow this guide.
 
-Ensure that you are using the **latest**, **stable** and **updated** version of the Operating System and Web Browser available to you on your machine.
+### Guidelines
 
-We appreciate any responsible disclosure of vulnerabilities that might impact the integrity of our platforms and users.
+We appreciate a responsible disclosure of vulnerabilities that might impact the integrity of our platforms and users. In the interest of saving everyone time, we encourage you to report vulnerabilities with these in mind:
```
Comment (ojeytonwilliams) [inline docs/security.md:12]:
```suggestion
We appreciate responsible disclosure of vulnerabilities that might impact the integrity of our platforms and users. In the interest of saving everyone's time, we encourage you to report vulnerabilities with these in mind:
```
Bit late, I know, but the 'a' doesn't belong here. Also `'s` is missing

Comment (nhcarrigan) [reply docs/security.md:12]: Not sure about the `'s`.

**Pull request merged**